app/viewmodels/checkAnswers/HasTradingNameSummary.scala

```diff
@@ -29,17 +29,16 @@ object HasTradingNameSummary  {
   def row(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
     for {
       hasTradingName        <- answers.get(HasTradingNamePage)
-      registeredCompanyName <- answers.get(RegisteredCompanyNamePage)
     } yield {
 
       val value = if (hasTradingName) "site.yes" else "site.no"
 
       SummaryListRowViewModel(
-        key     = messages("hasTradingName.checkYourAnswersLabel", registeredCompanyName),
+        key     = messages("hasTradingName.checkYourAnswersLabel"),
```

Comment:
If you have removed the parameter to the message does that message now show as {0} as I cant see a change to hasTradingName.checkYourAnswersLabel in here? same with the one below

Reply:
The messages no longer require a param so I don't think we need that here anymore.